bridge-stp.in

```diff
@@ -212,7 +212,7 @@ case "$action" in
             # Add bridge to mstpd and configure.
             echo "Adding/configuring bridge '$bridge' ..."
             "$mstpctl" addbridge "$bridge" || continue
-            if [ -n "$config_cmd" ]; then
+            if [ -x "$config_cmd" ]; then
```

Comment:
I vote 3.
I'd like to give the user the option to skip configuration entirely by setting $config_cmd to an empty string, so I'd rather not drop the check entirely.
I think something like this should have the desired effect:
if [ -x "$config_cmd" ] || [ "$(type -t "$config_cmd")" = 'function' ]; then